typings/index.d.ts

```diff
@@ -201,10 +201,10 @@ export type ChildrenFunction<Item> = (
   options: ControllerStateAndHelpers<Item>,
 ) => React.ReactNode
 
-export type DownshiftInterface<Item> = React.ComponentClass<
+export class Downshift<Item = any> extends React.Component<
```

Comment:
Curious if this could've been the following for backwards compatibility?

```suggestion
export type DownshiftInterface<Item = any> = React.ComponentClass<
```

Reply:
@NMinhNguyen I think we want to export `Downshift` accepting a generic `Item` type (as was not the case prior to this PR).  But we could probably export `DownshiftInterface` as well for backwards compatibility.  Maybe like `export type DownshiftInterface<Item> = Downshift<Item>` or something along those lines?  Thoughts?